package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
- [optional] I am naming this optional because the previous reviewer did not notice, but it is really required for your project to work. Great job removing the package.json duplicate. Please also ensure that you remove the dist, src folder and webpack.config.js file from the webpack folder into your root folder. Please fix that
![Screenshot (318)_LI](https://user-images.githubusercontent.com/65298048/186214032-7d0618db-78a7-485a-b96a-19e29a25c52f.jpg)